client/lib/domains/resolve-domain-status.js

```diff
@@ -45,12 +53,45 @@ export function resolveDomainStatus(
 			}
 
 			if ( ( ! isJetpackSite || isSiteAutomatedTransfer ) && ! domain.pointsToWpcom ) {
+				const status = translate(
+					'{{strong}}Verifying connection:{{/strong}} You can continue to work on your site, but you domain won’t be reachable just yet.',
+					{
+						components: {
+							strong: <strong />,
+						},
+					}
+				);
 				return {
-					statusText: translate( 'Complete setup' ),
-					statusClass: 'status-warning',
+					statusText: translate( 'Verifying connection' ),
+					statusClass: 'status-verifying',
+					icon: 'verifying',
+					listStatusText: status,
+					listStatusClass: 'verifying',
+				};
+			}
+
+			if ( hasConnectionError ) {
+				// New layout: "Connection error: We noticed that the name servers weren’t updated correctly, please try this step again."
+				const status = translate(
+					"{{strong}}Connection error:{{/strong}} We couldn't verify your connection. Please {{a}}follow this setup again{{/a}}.",
+					{
+						components: {
+							strong: <strong />,
+							a: (
+								<a
+									href={ domainManagementEdit( siteSlug, domain.domain ) }
+									onClick={ ( e ) => e.stopPropagation() }
+								/>
+							),
+						},
+					}
+				);
+				return {
+					statusText: translate( 'Connection error' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Publicize Connection Error' )` *ES Score: 7*